### PR TITLE
Fix info about SIGHUP

### PR DESCRIPTION
**Pull request by HardenedSteel**
SIGHUP doesn't reload config file

related to: https://github.com/PurpleI2P/i2pd/issues/384

### CODE REVIEW

**Comment by orignal** [comment]
No, it doesn't and shouldn't. It only reloads tunnels.

**Comment by HardenedSteel** [comment]
I think you misunderstood, current documentation says ``* HUP - Reload configuration files.`` but this sounds like it also reloads config file, that's why I changed to ``* HUP - Reload tunnel configuration file.`` which is easier to understand.

**Comment by orignal** [comment]
Where does it say this? I will fix.

File: docs/user-guide/run.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ i2pd can be controlled with signals. The process ID by is written to the file `~
 i2pd supports the following signals:
 
 * INT - Graceful shutdown. i2pd will wait for up to 10 minutes and stop. Send a second INT signal to shutdown i2pd immediately.
-* HUP - Reload configuration files.
```
Comment (HardenedSteel) [inline docs/user-guide/run.md:25]: @orignal

Comment (orignal) [reply docs/user-guide/run.md:25]: Thanks. Fixed.

**Pull request closed without merging**